<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>651</width>
      <height>401</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>–ù–∞–∑–≤–∞–Ω–∏–µ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>–ö–∞—Ç–µ–≥–æ—Ä–∏—è</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>–ù–æ–º–µ—Ä —Ç–µ–ª–µ—Ñ–æ–Ω–∞</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>–§–æ—Ç–æ</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>–í–≤–µ–¥–∏—Ç–µ –Ω–∞–∑–≤–∞–Ω–∏–µ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>–í—ã–±–µ—Ä–∏—Ç–µ –∫–∞—Ç–µ–≥–æ—Ä–∏—é:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_search">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>40</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>–õ—é–±–∞—è</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>–í–µ—â–∏</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>–û–¥–µ–∂–¥–∞</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>–ñ–∏–≤–æ—Ç–Ω—ã–µ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>–î—Ä—É–≥–æ–µ</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>47</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>22</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>–ü–æ–∏—Å–∫üîé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>–°–±—Ä–æ—Å–∏—Ç—å —Ñ–∏–ª—å—Ç—Ä—ã</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
